packages/common/src/validation.ts

```diff
@@ -9,9 +9,9 @@ const basePaths = Joi.array().items(basePath)
 
 const chain = Joi.object({
   namespace: Joi.string(),
-  id: Joi.alternatives()
-        .try(Joi.string().pattern(/^0x[0-9a-fA-F]+$/), 
-        Joi.number().positive()).required,
+  id: Joi.string()
+    .pattern(/^0x[0-9a-fA-F]+$/)
+    .required(),
```

Comment:
Any reason to change this validation?

Reply:
This was related to the [validation error fix for hardware wallets chain added to the main release: ](https://github.com/blocknative/web3-onboard/pull/1176/commits/7b1eb11799663711d3594217b85e49fcbabdb8dd)

There is another Joi.alternatives- string or number [validation included in the previous PR core chainID which .](https://github.com/blocknative/web3-onboard/pull/1135/files#diff-e8cb2da76902122de27c23ccd59551757403f6fba1a2fb3c272c47096166c507R26)

I guess placing chains: Chain[] | ChainWithDecimalId[] will allow for such validation but I'm not sure that's needed since the GetInterfaceHelpers used by Ledger works with the internal hex string.

Reply:
This seem correct if we are using Hex chain IDs internally(converting number chain ids to hex after initialization) it doesn't seem we would need positive number validation within the common package